src/main/java/seedu/ui/Ui.java

```diff
@@ -140,6 +141,8 @@ public void printReactionToRemovingItem(Item item, int qty) {
      * Suggests the correct item name if input is incomplete.
      *
      * @param predictedItem Only item available containing search term but is not exactly the same.
+     * @return True if the user said 'y' or False if the User said 'n'
```

Comment:
this is missing a full stop might be the cause of checkstyle error

Reply:
Thanks for noticing, wasn't the cause of the error, but I almost forgot to put the period in.